MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -401,6 +401,11 @@
 				shaderIntFuncsFeatures->shaderIntegerFunctions2 = true;
 				break;
 			}
+			case VK_STRUCTURE_TYPE_PHYSICAL_DEVICE_SHADER_ATOMIC_FLOAT_FEATURES_EXT: {
+				auto* atomicFloatFeatures = (VkPhysicalDeviceShaderAtomicFloatFeaturesEXT*)next;
```

Comment:
We can't assume the app has cleared the `VkPhysicalDeviceShaderAtomicFloatFeaturesEXT` before submitting it. Please either explicitly set the values of all the flags, or add the following line after this line, and before setting the `shaderBufferFloat32AtomicAdd` flag:

`mvkClear(atomicFloatFeatures);`